innopoints/views/application.py

```diff
@@ -6,8 +6,11 @@
 - PATCH  /projects/{project_id}/activities/{activity_id}/applications/{application_id}
 
 VolunteeringReport:
-- GET  /projects/{project_id}/activities/{activity_id}/applications/{application_id}/report_info
-- POST /projects/{project_id}/activities/{activity_id}/applications/{application_id}/report
+- GET    /projects/{project_id}/activities/{activity_id}/applications/{application_id}/report_info
+- GET    /projects/{project_id}/activities/{activity_id}/applications/{application_id}/report
```

Comment:
I do not see this endpoint defined anywhere. In line 371, only POST, PATCH, and DELETE are defined. Frankly, it makes more sense than "report_info" (in terms of naming). I would suggest either:
- Remove this line
- Add it as an alias as to not break existing frontend code
- Move the current "report_info" endpoint into the new `VolunteeringReportAPI` class, keeping this line and removing the previous one (recommended. requires changing one line in frontend)

Reply:
Shit, I forgot to remove this line. Originally I had that endpoint, but then deleted it because it was useless. As for `/report_info`, it serves a different purpose, aggregating all reports and providing an average rating, so it wouldn't feel right in that class.